lte/gateway/c/oai/lib/n11/SubscriberdbServiceClient.cpp

```diff
@@ -0,0 +1,141 @@
+/**
```

Comment:
the name of the file should be reflective of the specific service being consumed from the subscriberdb, e.g., M5GAuthenticationServiceClient.cpp.

Reply:
Comments addressed.